app/controllers/pets/applications_controller.rb

```diff
@@ -5,9 +5,16 @@ def show
   end
 
   def create
-    application = Application.create!(create_params)
-    application.save
-    redirect_to "/applications/#{application.id}"
+    application = Application.new(create_params)
+    # binding.pry
+    if application.save
+      redirect_to "/applications/#{application.id}"
+    else
```

Comment:
Let's ask about this during out checkin. No point staying stuck on this for ages.